content/master/concepts/compositions.md

```diff
@@ -0,0 +1,993 @@
+---
+title: Compositions
+weight: 30
+aliases: 
+  - composition
+---
+
+Compositions are a template for creating multiple managed resources as a single
+object. 
+
+A Composition _composes_ individual managed resources together into a larger,
+reusable, solution.
+
+An example Composition may combine a virtual machine, storage resources and
+networking policies. A Composition template links all these individual
+resources together. 
+
+{{<expand "Confused about Compositions, XRDs, XRs and Claims?" >}}
+Crossplane has four core components that users commonly mix up:
+
+* Compositions - This page. A template to define how to create resources.
+* [Composite Resource Definition]({{<ref "./composite-resource-definitions">}})
+  (`XRD`) - A custom API specification. 
+* [Composite Resource]({{<ref "./composite-resources">}}) (`XR`) - Created by
+  using the custom API defined in a Composite Resource Definition. XRs use the
+  Composition template to create new managed resources. 
+* [Claims]({{<ref "./claims" >}}) (`XRC`) - Like a Composite Resource, but
+  with namespace scoping. 
+{{</expand >}}
+
+## Creating Compositions
+
+Creating Compositions consists of:
+* [Resource templates](#resource-templates) defining the resources to create.
+* [Enabling Composite Resources](#enabling-composite-resources) to use this
+  Composition template.
+
+Optionally, Compositions also support:
+* [Modifying and patching](#changing-resource-fields) resource settings.
+* [Storing connection details](#storing-connection-details) and secrets
+  generated by the managed resources.
+* Using [Composite functions](#composite-functions) to allow custom programs to
+  run alongside the Composition. 
+* Creating a 
+  [custom definition of when a resource is ready](#resource-readiness-checks) to use. 
+ 
+###  Resource templates
+The 
+{{<hover label="resources" line="4">}}resources{{</hover>}} field of a 
+Composition's 
+{{<hover label="resources" line="3">}}spec{{</hover>}} 
+defines the set of things that a Composite Resource can create.
```

Comment:
will is future tense and isn't used in the docs.
"should" implies "maybe it will, maybe it won't, let's hope it works 🤞"

Using "can" puts it in the context of expected, present tense, behavior.